stackslib/src/chainstate/stacks/miner.rs

```diff
@@ -2714,9 +2753,18 @@ impl BlockBuilder for StacksBlockBuilder {
                                             100 - TX_BLOCK_LIMIT_PROPORTION_HEURISTIC,
                                             &total_budget
                                         );
+                                    let mut measured_cost = cost_after;
```

Comment:
Why declare `mut measured_cost`?  Can't you just use `cost_after.sub(&cost_before)`?

Reply:
I just like the name `measured_cost` more -- once I've mutated the value, its not really "the cost after" anymore.